src/components/Buttons/ButtonPurpleApprove.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+const ButtonPurpleApprove = () => {
+  return <StyledButton type="button">Approve</StyledButton>;
```

Comment:
Following the comment about button variant, you can add some logic that if buttonVariation === ButtonPurpleApprove, then title = "Approve" :)